playbook/ranger/config/ranger-ozone-audit.xml

```diff
@@ -0,0 +1,207 @@
+	====
+
+	<property>
+		<name>xasecure.audit.destination.hdfs</name>
+		<value>enabled</value>
+	</property>
+
+	<property>
+		<name>xasecure.audit.destination.hdfs.dir</name>
+		<value>wasb://ranger-audit1@youraccount.blob.core.windows.net</value>
+	</property>
+
+	the following 3 correspond to the properties with similar name in core-site.xml, i.e.
+	- fs.azure.account.key.youraccount.blob.core.windows.net => xasecure.audit.destination.hdfs.config.fs.azure.account.key.youraccount.blob.core.windows.net and
+	- fs.azure.account.keyprovider.youraccount.blob.core.windows.net => xasecure.audit.destination.hdfs.config.fs.azure.account.keyprovider.youraccount.blob.core.windows.net,
+	- fs.azure.shellkeyprovider.script => xasecure.audit.destination.hdfs.config.fs.azure.shellkeyprovider.script,
+
+	<property>
+		<name>xasecure.audit.destination.hdfs.config.fs.azure.account.key.youraccount.blob.core.windows.net</name>
+		<value>YOUR ENCRYPTED ACCESS KEY</value>
+	</property>
+
+	<property>
+		<name>xasecure.audit.destination.hdfs.config.fs.azure.account.keyprovider.youraccount.blob.core.windows.net</name>
+		<value>org.apache.hadoop.fs.azure.ShellDecryptionKeyProvider</value>
+	</property>
+
+	<property>
+		<name>xasecure.audit.destination.hdfs.config.fs.azure.shellkeyprovider.script</name>
+		<value>/usr/lib/python2.7/dist-packages/hdinsight_common/decrypt.sh</value>
+	</property>
+	-->
+
+	<!-- Log4j audit provider configuration -->
+	<property>
+		<name>xasecure.audit.log4j.is.enabled</name>
+		<value>false</value>
+	</property>
+
+	<property>
+		<name>xasecure.audit.log4j.is.async</name>
+		<value>false</value>
+	</property>
+	<property>
+		<name>xasecure.audit.log4j.async.max.queue.size</name>
+		<value>10240</value>
+	</property>
+
+	<property>
+		<name>xasecure.audit.log4j.async.max.flush.interval.ms</name>
+		<value>30000</value>
+	</property>
+	<!-- Ozone audit provider configuration -->
+	<property>
+		<name>xasecure.audit.ozone.is.enabled</name>
+		<value>false</value>
+	</property>
+
+	<property>
+		<name>xasecure.audit.ozone.async.max.queue.size</name>
+		<value>1</value>
+	</property>
+
+	<property>
+		<name>xasecure.audit.ozone.async.max.flush.interval.ms</name>
+		<value>1000</value>
+	</property>
+	<property>
+		<name>xasecure.audit.ozone.broker_list</name>
+		<value>localhost:9092</value>
+	</property>
+
+	<property>
+		<name>xasecure.audit.ozone.topic_name</name>
+		<value>ranger_audits</value>
+	</property>
+	<!-- Ranger audit provider configuration -->
+	<property>
+		<name>xasecure.audit.solr.is.enabled</name>
+		<value>false</value>
+	</property>
+	<property>
+		<name>xasecure.audit.solr.async.max.queue.size</name>
+		<value>1</value>
+	</property>
+
+	<property>
+		<name>xasecure.audit.solr.async.max.flush.interval.ms</name>
+		<value>1000</value>
+	</property>
+	<property>
+		<name>xasecure.audit.solr.solr_url</name>
+		<value>http://localhost:6083/solr/ranger_audits</value>
```

Comment:
Ignore that as well. We won't be using solr anytime soon.

Reply:
Removed.